src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DeleteAccount/Components/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import * as S from '../styles';
+import * as C from '../../../ignitus-Atoms/colors';
+import { withErrorBoundary } from '../../../../ignitus-ErrorHandlingComponents/errorBoundary';
+import { Heading1 } from '../../../..';
+import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
+import { SM } from '../../../ignitus-Atoms/fonts';
+
+export const DeleteAccount = withErrorBoundary(() => (
+  <S.DeleteContainer>
+    <S.TopSection>
+      <Heading1>Delete Account</Heading1>
+      <S.IconContainer color={C.White}>
+        <S.StyledIcon name={AppIcon.CrossIcon} color={C.GreyOne} />
+      </S.IconContainer>
+    </S.TopSection>
+    <S.MiddleSection>
+      <S.StyledParagraph>
+        We were hoping this day would never come, Sophia. Before we proceed,
```

Comment:
Dynamic username would be useful. What you say?

Reply:
@Marvin9 added `username` as props